src/components/TracePage/TracePageHeader.js

```diff
@@ -104,7 +104,12 @@ export default function TracePageHeader(props: TracePageHeaderProps) {
               <Dropdown.Menu>
                 <Dropdown.Item>
                   <a rel="noopener noreferrer" target="_blank" href={`/api/traces/${traceID}`}>
```

Comment:
@tiffon I don't see this file in #151, it probably needs adjustment with the prefix as well

Reply:
@yurishkuro Yes, this needs the `prefixUrl(...)`.

Also, is the trailing `/` in the raw version URL intentional?

```
/api/traces/${traceID}/?raw=true
```

Seems like the only difference should be the query string is appended?

```
/api/traces/${traceID}?raw=true
```